docs/user/arena-setup.rst

```diff
@@ -2,269 +2,272 @@
+- **Class selection signs**: Clicked to select a class
+- **Iron block**: Clicked to mark self as "ready"
+
+Class selection signs must have the name of the class on the first line
+(case-sensitive). The last three lines are not checked and they can have any
+text. You will not receive a confirmation message after making a new sign since
+MobArena does not register the creation of class selection signs.
+
+Arena floor
+===========
+
+An arena floor is where the action happens. Players fight through mob waves on
+the arena floor. How the arena floor looks is up to you, but we recommend these
+minimum requirements:
+
+- Closed in by walls on all sides
+- Have a ceiling / roof
+
+This prevents players and mobs from escaping and also blocks players from
+wandering out of the arena with class items.
+ander out of the arena.
+
+Spectator area
+==============
+
+A spectator area lets non-players watch an on-going match. The ``/ma spec``
+command teleports a player into the spectator area. If configured, [#]_ players
+warp into the spectator arena when they die.
+
+Design the area so spectators cannot escape the spectator area, since they
+invincible. Spectators should *not* enter the arena floor or exit the spectator
+area on foot.
+
+.. [#] Set ``spectate-after-death`` to ``true`` in the config file to force
+   players to the spectator area after dying
+
+Exit point
+==========
+
+When players leave the arena or when the last player standing dies, arena
+players and spectators teleport to the location they joined from. Optionally, an
+arena can have an exit warp. This controls where players go after leaving a
+match.
+
+
+****************
+Defining regions
+****************
+
+When an arena is ready, it must be **defined** into…
+
+#. Regions
+#. Warps
+#. Spawnpoints
+#. Containers
+
+Regions
+=======
+
+Arenas **must** have an arena region and optionally a lobby region. Regions are
+set using the Regions tools. MobArena uses the arena region to…
+
+- Stop cheating by kicking players if they leave the arena region
+- Only spawn MobArena mobs inside arena region
+
+Warps
+=====
+
+Players teleport to different warps for different events. There are four warps
+used in MobArena:
+
+- **Lobby warp**: Warp location for players joining a new match; leave when all
+  players "ready up" or match countdown timer ends
+- **Arena warp**: Warp location for players to spawn in the arena floor
+- **Spectator warp**: Warp location for spectators to watch an on-going match
+- **Exit warp**: Optional warp location for players to teleport to after a match
+  finishs or they leave the arena
+
+Spawnpoints
+===========
+
+Mobs spawn at the spawnpoint(s) of an area. MobArena only uses spawnpoints in a
+*15-block radius* from any player. An arena can have multiple spawnpoints.
+Spawnpoints are added using the Spawnpoints tool.
+
+The number of mobs spawned is not determined by the number of spawnpoints, but
+actual formulas. See :doc:`wave-formulas` for more information.
```

Comment:
This link shows up a bit weird on github - is that normal? It also seems to not link (on github) to anything, but I guess that may just be part of the pull request thingie.

Reply:
@garbagemule This is an internal citation to another document. In this case, it links to the `Wave formulas` docs. It won't render on GitHub, but in Sphinx, it correctly links it to the right document. It took me a little bit to figure this one out and get it working. Some of the docs are [here](http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-documents), if you're interested. 😄